src/helpers/json_helper.py

```diff
@@ -15,3 +16,27 @@ def write_json_file(file_path: str, content: dict | list):
     """
     with open(file_path, "w", encoding="UTF-8") as file:
         dump(content, file, indent=4)
+
+
+def read_json_file(file_path: str) -> dict | list:
+    """
+    This will read the content from the json file at the specified path.
+
+    Args:
+        file_path (str): The path to the json file.
+
+    Returns:
+        dict | list: Returns content of json file.
+
+    Raises:
+        FileNotFoundError: If the file does not exist.
+        ValueError: If the file is not a valid json file.
+    """
+    try:
+        with open(file_path, "r", encoding="UTF-8") as file:
+            return load(file)
+    except JSONDecodeError:
+        raise ValueError(f"File at {file_path} is not a valid json file.")
```

Comment:
Pylint had a issue with this, I dont actually know myself what it means but I just know it wants us to say

```python
except JSONDecodeError as exc:
        raise ValueError(f"File at {file_path} is not a valid json file.") from exc
```

if you have time you please do some research on this, teach me something new :)